src/js/setup-audio-tracks.js

```diff
@@ -35,10 +35,30 @@ function handlePlaybackMetadataLoaded(player, tech) {
   const currentAudioTrack = mediaPlayer.getCurrentTrackFor('audio');
 
   dashAudioTracks.forEach((dashTrack) => {
-    let label = dashTrack.lang;
+    let localizedLabel;
 
-    if (dashTrack.roles && dashTrack.roles.length) {
-      label += ` (${dashTrack.roles.join(', ')})`;
+    if (Array.isArray(dashTrack.labels)) {
+      for (let i = 0; i < dashTrack.labels; i++) {
+        if (player.language().includes(dashTrack.labels[i].lang.toLowerCase())) {
```

Comment:
Sorry I didn't catch this before, but `includes` doesn't exist in IE11 :sob:

Reply:
I think you should be able to get away with just using `indexOf() > -1`

Reply:
more common to use `indexOf() !== -1`